zombie-tests/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "zombie-tests",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {},
+  "dependencies": {
+    "@polkadot-api/cli": "workspace:*",
+    "@polkadot-api/client": "workspace:*",
+    "@polkadot-api/sc-provider": "workspace:*",
+    "@polkadot-api/substrate-bindings": "workspace:*",
+    "@polkadot-api/substrate-client": "workspace:*",
+    "@polkadot-api/substrate-codegen": "workspace:*",
+    "@polkadot-api/utils": "workspace:*",
+    "typescript": "^5.1.6"
+  },
+  "devDependencies": {
+    "@types/node": "^20.6.0",
+    "prettier": "^2.6.2"
+  },
+  "prettier": {
+    "printWidth": 80,
+    "semi": false,
+    "trailingComma": "all"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
```

Comment:
`MIT`

Reply:
for consistency with the other packages let's leave it as `MIT` and then address `GPL-3.0-only` in a different PR